### ISSUE TITLE
Issues and PRs seem to randomly disconnect

### ISSUE THREAD

**Issue by danielbartsch** [ZenHubIO/support#1585]
Me and my team noticed for the last days, that it sometimes happens that issues disconnect from PRs. This is very annoying.

We could not reproduce it. And also there is no log in the issue or PR about a disconnection.

This happens on a private repo.

**Comment by javjim-mautic** [comment]
only time this happens to me is when the ticket gets closed

**Comment by danielbartsch** [comment]
Yes I know this case. It has always been this way, and it's expected. But this issue is about PRs and Issues that get disconnected even if they are not closed.

**Comment by chemitaxis** [comment]
Same here... the PR connections is very strange, sometimes if you open again an issue that was closed, the PR is connected again and sometimes the PR is still closed... very frustrating...

**Comment by Startouf** [comment]
Same here, this has been going on for quite some tim and it's getting really annoying. 
Possible leads 
- PR was force pushed
- Breaking change from github (since the Microsoft acquisition they've been shipping a lot of features)
- May occur when moving issues on the board

**Comment by Startouf** [comment]
Sorry, but the problem is getting worse here in our organization private repo we are paying for, and issues now seem to disconnect almost whenever a commit is pushed

**Comment by chemitaxis** [comment]
I think the problem is with the webhooks, I’m speaking with support... and my webhooks has deleted suddenly. I’m waiting for a solution or moving to other option. The problem started when GitHub was down...

**Comment by chemitaxis** [comment]
My problem was solved. Send an email to support, they reset my webhooks and now it works like a charm.
